manifest_staging/charts/aad-pod-identity/README.md

```diff
@@ -291,6 +291,7 @@ The following tables list the configurable parameters of the aad-pod-identity ch
 | `rbac.enabled`                            | Create and use RBAC for all aad-pod-identity resources                                                                                                                                                                                                                                                                        | `true`                                                         |
 | `rbac.pspEnabled`                         | If `true`, create and use a restricted pod security policy for AAD Pod Identity pod(s)                                                                                                                                                                                                                                        | `false`                                                        |
 | `rbac.allowAccessToSecrets`               | NMI requires permissions to get secrets when service principal (type: 1) is used in AzureIdentity. If using only MSI (type: 0) in AzureIdentity, secret get permission can be disabled by setting this to false.                                                                                                              | `true`                                                         |
+| `rbac.createUserFacingClusterRoles` | If set to true, then view and edit cluster roles will be created with annotations that agrigate to the admin, edit and view built-in Kubernetes cluster roles. These roles will be able to create the necessary resources to allow pod identity binding on pods. | `true` |
```

Comment:
```suggestion
| `rbac.createUserFacingClusterRoles` | If set to true, then view and edit cluster roles will be created with annotations for the admin, edit and view built-in Kubernetes cluster roles. These roles will be able to create the necessary resources to allow pod identity binding on pods. | `false` |
```

Reply:
The default value should be `false`

Reply:
Thanks it was just set to true in the docs, fixed.